rackcity/views/rackcity_utils.py

```diff
@@ -128,10 +128,13 @@ def validate_asset_location(
 
 def validate_location_modification(data, existing_asset, user, change_plan=None):
     asset_id = existing_asset.id
-    rack_id = existing_asset.rack.id
+    rack_id = existing_asset.rack_id
```

Comment:
what's this? I don't think `rack_id` is a field on Asset?

Reply:
so django actually stores the rack foreign key as rack_id, so this is apparently are more efficient way to access the rack id . this goes for all foreign field keys

Reply:
*fields

Reply:
you learn something new every day